misc/services/mobile-webui/mobile-webui-frontend/src/components/ScanHUAndGetQtyComponent.jsx

```diff
@@ -98,7 +98,7 @@ const ScanHUAndGetQtyComponent = ({
     // noinspection UnnecessaryLocalVariableJS
     const resolvedBarcodeDataNew = {
       ...resolvedBarcodeData,
-      ...resolveScannedBarcode?.(scannedBarcode),
+      ...resolveScannedBarcode?.(scannedBarcode, handlingUnitInfo),
```

Comment:
hmmm,  
i think you shall convert those resolveScannedBarcode functions to accept object params, i.e.
resolveScannedBarcode?.({ scannedBarcode, handlingUnitInfo })

More, i would send only the huId and not the whole block because it seems only the huId is used.

Reply:
moved to using just the huId, but since the component is used a lot, I've skipped doing the other change to minimize the risk of screwing something up